app/views/items/_browse_facet_header.html.erb

```diff
@@ -0,0 +1 @@
+<h1><%= t "browse.browse_type", default: "Browse by" %> <%= @browse_facet_info["label"] %></h1>
```

Comment:
I think this one should be a lower level heading. Probably an `<h2>`. Because this is the header for the nav on the left side that is followed by the site / section's facets, right? `<h1>` should just be for the overall page titles

Reply:
It's _not_ the header I thought it was. Nevermind